test/test_two_links_moving_joint.cpp

```diff
@@ -0,0 +1,88 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Is code moved/copied from somewhere or should this copyright line be updated?

Reply:
It's basically ported from the ROS 1 test in robot_state_publisher [here](https://github.com/ros/robot_state_publisher/blob/fe661c1503cad7fedc8c16be6edddb5eb18d5a65/test/test_two_links_moving_joint.cpp), so I thought the copyright should remain the same.